pkg/services/dashboards/service/search/search.go

```diff
@@ -99,6 +101,10 @@ func ParseResults(result *resource.ResourceSearchResponse, offset int64) (*v0alp
 		}
 		if folderIDX > 0 && row.Cells[folderIDX] != nil {
 			hit.Folder = string(row.Cells[folderIDX])
+			if hit.Resource == v0alpha1.RESOURCE && hit.Folder != "" && folderSet[hit.Folder] == false {
```

Comment:
Looks like this isn't being used anywhere after being set

Reply:
good catch meant to remove that